website/docs/r/vapp_vm.html.markdown

```diff
@@ -78,4 +84,11 @@ The following arguments are supported:
 * `power_on` - (Optional) A boolean value stating if this vApp should be powered on. Default is `true`
 * `accept_all_eulas` - (Optional; *v2.0+*) Automatically accept EULA if OVA has it. Default is `true`
 * `org` - (Optional; *v2.0+*) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
-* `vdc` - (Optional; *v2.0+*) The name of VDC to use, optional if defined at provider level
\ No newline at end of file
+* `vdc` - (Optional; *v2.0+*) The name of VDC to use, optional if defined at provider level
+* `disk` - (Optional) Independent disk attachment configuration. Details below
```

Comment:
Tag that this param is from 2.1+ missing.

Reply:
Fixed